tools/dump_tool/HowToUseDumpTool.md

```diff
@@ -272,6 +272,35 @@ python dumpParser.py -r $TIZENRT_BASEDIR/build/output/bin/ramdump_0x02020000_0x0
 ```
 2. See the Output
 
+#### To display Debug Symbols
+1. Copy crash logs
+    First copy the crash logs to a file named log_file in tools/dump_tool/`<log_file>`
+2. Run Script in tools/dump_tool path
+    $ python3 ramdumpParser.py -e `<Tinyara elf path>` -t `<Log file path>`
+
+    ex)
+    $ python3 ramdumpParser.py -e ../../build/output/bin/tinyara.axf -t ./log_file
+
+Example Call Stack Output
+```
+*************************************************************
+dump_file         : None
+log_file          : .log_file
+Tinyara_elf_file  : ../../build/output/bin/tinyara.axf
```

Comment:
Most of the other boards uses `tinyara` which is elf format.
Only rtl8721csm uses `.axf` format.
So I think it is better to guide to use `tinyara` instead of `tinyara.axf`.